src/vs/editor/common/editorCommon.ts

```diff
@@ -4164,6 +4167,33 @@ export var EventType = {
 };
 
 /**
+ * Positions on the view
+ */
+export const ViewPosition = {
```

Comment:
I suggest renaming `ViewPosition` to `MoveCommandViewPosition` or something more specific -- for someone looking at `monaco.d.ts` the current name might be very confusing, as it is simply typing an argument to a specific command and is not a new concept of `Position` <-> `ViewPosition`.